src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -9,6 +9,7 @@ import RenderQueue from './Platform/Circuits/RenderQueue';
 import { ConsoleLogger } from './Platform/Logging/Loggers';
 import { LogLevel, ILogger } from './Platform/Logging/ILogger';
 import { discoverPrerenderedCircuits, startCircuit } from './Platform/Circuits/CircuitManager';
+import { setRendererEventDispatcher } from './Rendering/Renderer';
```

Comment:
The `setRendererEventDispatcher` mechanism seems very over-complicated to me. In the end, it loops over a `const` colllection of `BrowserRender` instances and calls a method on them to change a property value. Since this collection is backed by a `const` anyway, it's exactly equivalent to having a fixed global value for `rendererEventDispatcher`.

So as a simpler but functionally equivalent alternative, could we have a shared global `rendererEventDispatcher` function that `BrowserRenderer` calls directly?

Reply:
done